src/pad/padPane.ts

```diff
@@ -190,16 +192,38 @@ const paneDef: PaneDefinition = {
       }
     }
 
+    /**   Button to Export the pad to HTML
+    */
+    var exportHTMLButton = function (subject) {
+      const copyIcon = icons.iconBase + 'noun_681601.svg'
+      return widgets.button(dom, copyIcon, 'Copy as HTML',
+        async function (_event) {
+          const htmlText = pad.notePadToHTML(subject, store)
```

Comment:
Might need a lower-case `p` if we merge https://github.com/solid/solid-ui/pull/211/commits/a79b324bb67e9630431520ea55e261e2770ec15d